pkg/cmd/image/list.go

```diff
@@ -128,6 +130,46 @@ type imagePrintable struct {
 
 func printImages(ctx context.Context, client *containerd.Client, imageList []images.Image, options *types.ImageListOptions) error {
 	w := options.Stdout
+	var ImageList []images.Image
```

Comment:
That is not a good name.

It should not start with a cap letter, and is very confusing to look at while the arg is `imageList`.

Suggesting instead `finaleImageList`? or anything else you like?